api/v1/runtime/common.proto

```diff
@@ -42,6 +42,7 @@ message Tuple {
   string dst_ip = 2;
   uint32 src_port = 3;
   uint32 dst_port = 4;
+  string dns_question = 5;
```

Comment:
How do we handle cases where we observe multiple domain names getting resolved to the same IP? 
I guess we need to switch this to a `repeated string`.

Reply:
I now store simple key pair cache of IP to DNS question string. I think in 99% of cases this will be a single entry.
Changing it to slice will complicate dns cache side and increase message size.
Worst case we will report some alias dns for this ip.

Reply:
As discussed in PM, it is fine to stick to a single DNS name, as this is anyway just best effort 👌